README.md

```diff
@@ -3,15 +3,17 @@ This project is about recreating our own version of bash with limited functional
 A miniature shell :).
 
 ## Valgrind
-```valgrind -s --leak-check=full --show-reachable=yes --error-limit=no --suppressions=minishell.supp ./minishell```
+```valgrind -s --leak-check=full --show-reachable=yes --error-limit=no --suppressions=minishell.supp --trace-children=yes --track-fds=yes ./minishell```
```

Comment:
Finally added it here instead of always adding it to the compilation manually :)
But I'm not sure that we are actually closing all (and only) the fds that we should. There is also another way of checking this, by running a terminal command like ```ls -l /proc/$$/fd``` or similar that Google returns.